app/bundles/LeadBundle/Tests/Controller/Api/LeadApiControllerFunctionalTest.php

```diff
@@ -198,7 +198,7 @@ public function testBachdDncAddAndRemove()
         $clientResponse = $this->client->getResponse();
         $response       = json_decode($clientResponse->getContent(), true);
 
-        $this->assertSame(null, $response['contacts'][0]['doNotContact'][0]['reason']);
+        $this->assertFalse(isset($response['contacts'][0]['doNotContact'][0]));
```

Comment:
Do you know why this had to be changed? I fear it may cause issues elsewhere.

Reply:
Yes I\ know. I tested it manually and the record is missing after action above. Thus there is no DNC record present. I testedd it through my branch, staging and master and the behavior is the same. It is only present on 7.2 ans NOTICE during functional testing. The other PHP versions was fine.

You can find it on my other PRs too.